### PR TITLE
feat(redux-persist-profile): implement redux persist into profile page

### LINKED ISSUE #68: Implement redux persist into Login-Page
**Is your feature request related to a problem? Please describe.**
Need redux persist to be added to profile page so that the user can stay logged in.

**Describe the solution you'd like**
Give the user the ability to have their information saved for their profile.

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by JorgeLVilla**
## Changes
_List Changes Introduced by this PR_
1. Installed Redux Persist 
2. Installed React-Native-Async-Storage
3. The user is now able to save their profile information.
4. Converted images into Base64 files so that they can be stored in redux.

## Purpose
This allows the user to have their name and avatar profile picture saved locally.

_If this closes an issue, name it here. If it doesn't, remove this line_
Closes #68

### CODE REVIEW

**Review by Gjhnsn — APPROVED** [review]
persisting data, nice!

File: App.js [hunk 1]
```diff
@@ -1,15 +1,17 @@
...
-import store from './redux/store';
+import {store, persistor }  from './redux/store';
+import { PersistGate } from 'redux-persist/integration/react';
 import { LevelSelect, MainMenu, ProfilePage } from './components';
 import randomGameSelect from './utils/randomGameSelect';
 
 export default function App() {
   return (
     <Provider store={store}>
+      <PersistGate loading={null} persistor={persistor}>
```
Comment (connielion) [inline App.js:14]: noice

**Review by connielion — APPROVED** [review]
Looks great overall!

**Review by JayDampitan — APPROVED** [review]
Looks good Jorge! save that game.

**Review by dannysanchez559 — APPROVED** [review]
cool, looks like you guys got your persist working?

**Pull request merged**